packages/form-state-manager/src/types/field-config.ts

```diff
@@ -0,0 +1,10 @@
+import AnyObject from './any-object';
```

Comment:
this file should have `.d.ts` extension. Seems like the old one had the wrong extension as well.